elixir/apps/domain/lib/domain/resources.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Domain.Resources do
   alias Domain.{Repo, Validator, Auth}
   alias Domain.Resources.{Authorizer, Resource}
+  alias Domain.Gateways
```

Comment:
This is not very important but the convention is to first `alias` generic things like `Repo` and third-party modules, then other domain contexts, and after then stuff within the current context. This puts code dependencies to the top and makes it easier to track if the module violates the boundaries too much.
```suggestion
  alias Domain.Gateways
  alias Domain.Resources.{Authorizer, Resource}
```

Reply:
Good to know.  Appreciate the insight.